src/CTA.WebForms2Blazor/FileConverters/ConfigFileConverter.cs

```diff
@@ -3,19 +3,57 @@
 using System.Text;
 using System.Threading.Tasks;
 using CTA.WebForms2Blazor.FileInformationModel;
+using CTA.Rules.Actions;
+using CTA.Rules.Models;
+using Newtonsoft.Json.Linq;
+using System.Configuration;
+using System.IO;
+using System.Reflection;
+using Newtonsoft.Json;
+using System.Linq;
 
 namespace CTA.WebForms2Blazor.FileConverters
 {
     public class ConfigFileConverter : FileConverter
     {
+        private const string webConfigFile = "web.config";
+        private string _projectPrefix;
+        private string _fullPath;
+        private string _relativeDirectory;
+
         public ConfigFileConverter(string relativePath) : base(relativePath)
         {
+            var workingDirectory = Environment.CurrentDirectory;
+            _projectPrefix = Directory.GetParent(workingDirectory).Parent.Parent.FullName;
+            _fullPath = Path.Combine(_projectPrefix, RelativePath);
+            _relativeDirectory = Path.GetDirectoryName(RelativePath);
 
         }
 
-        public override Task<IEnumerable<FileInformation>> MigrateFileAsync()
+
+        public override async Task<IEnumerable<FileInformation>> MigrateFileAsync()
         {
-            throw new NotImplementedException();
+            string filename = Path.GetFileName(RelativePath);
+            FileInformation fi;
+
+            if (filename.Equals(webConfigFile))
+            {
+                //ProjectType WebForms doesn't really exist yet, but can be added for more specific configuration
+                ConfigMigrate configMigrate = new ConfigMigrate(_fullPath, ProjectType.WebForms);
```

Comment:
This only handles 1 config file but WebForms projects can have multiple Web.Config files, are you planning to implement multi-file support later?

Reply:
Well the name should still be web.config, but they may be in different modules. This should convert each web.config file seperately but it won't merge them together. I'm not sure what the expected behavior is tbh, are multiple web.config files supposed to end up all in one appsettings.json file are should there be multiple appsettings.json files as well

Reply:
I'm pretty sure there should only be 1 appsettings.json file

Reply:
Staying in line with my comment above, let's aim to process just a single web.config, at least to start.